src/commands/clear-account.js

```diff
@@ -0,0 +1,188 @@
+const { getAWSSDK } = require("../lib/aws");
+const { Command, flags } = require("@oclif/command");
+const { checkVersion } = require("../lib/version-check");
+require("colors");
+const inquirer = require("inquirer");
+const { getAllLambdasCount, deleteAllLambdas } = require("../lib/lambda");
+const { getAllRolesCount, deleteAllRoles } = require("../lib/iam");
+const { getAllApiGwCount, deleteAllApiGw } = require("../lib/apigw");
+const { getBucketCount, deleteAllBuckets } = require("../lib/s3");
+const { deleteAllStacks, getAllStacksCount } = require("../lib/cloudformation");
+const { getCurrentProfile } = require("../lib/utils");
+
+class ClearAccountCommand extends Command {
+	async run() {
+		const { flags } = this.parse(ClearAccountCommand);
+		const { force, profile } = flags;
+
+		global.profile = profile;
+
+		checkVersion();
+		const AWS = getAWSSDK();
+		if (force) {
+			this.log("Forcing account cleanup!".red.bold);
+			await this.clearEnvironment(AWS);
+		} else {
+			const profileName = getCurrentProfile();
+			const sts = new AWS.STS();
+			const caller = await sts.getCallerIdentity().promise();
+
+			const message = `You are about clear account [${
+				caller.Account
+			}] while using [${
+				profileName ? profileName : "unknown"
+			}] profile, are you sure?`;
+			const { clear } = await inquirer.prompt([
+				{
+					type: "confirm",
+					name: "clear",
+					message: message,
+					default: false
+				}
+			]);
+			if (clear) {
+				await this.clearEnvironment(AWS);
+			} else {
+				this.log("Not clearing environment");
+			}
+		}
+	}
+
+	_summary(results, singularName, hasRegion) {
```

Comment:
`_` prefix doesn't make the function private, and it's not a convention that we have adopted in this project anyway, so let's be internally consistent